yolort/v5/models/yolo.py

```diff
@@ -102,6 +101,7 @@ def _make_grid(self, nx=20, ny=20, i=0):
         return grid, anchor_grid
 
 
+@_dependency.requires_module("thop")
```

Comment:
Also we suggest to move `@_dependency.requires_module("thop")` before the `self._profile_one_layer()` method here.

https://github.com/zhiqwang/yolov5-rt-stack/blob/bf05b4b510b2a41ce4d8975b9b1be33784d29450/yolort/v5/models/yolo.py#L215